sdk/communication/communication-administration/test/communicationIdentityClientWithToken.spec.ts

```diff
@@ -4,15 +4,15 @@
 import { assert } from "chai";
 import { Recorder } from "@azure/test-utils-recorder";
 import { CommunicationIdentityClient } from "../src";
-import { createRecordedCommunicationIdentityClientWithToken } from "./utils/recordedClient";
```

Comment:
Why this change is necessay for the "COMMUNICATION_ENDPOINT" rename to 
      "COMMUNICATION_ENDPOINT_STRING"?

Reply:
it's not necessary but .net uses COMMUNICATION_ENDPOINT_STRING in their test-resources.json so just being consistent

Reply:
I think Reza just removed the _STRING part in .NET, so I wouldn't add it here.

Reply:
I prefer COMMUNICATION_ENDPOINT for test-resources.json, the string suffix for ConnectionString is part of the name, and is not supposed to indicate the type.